exec/Chainweb/Lookups.hs

```diff
@@ -74,8 +76,8 @@ headersBetween env (cid, Low low, High up) = do
     f :: T.Text -> Maybe BlockHeader
     f = hush . (B64.decode . T.encodeUtf8 >=> runGet decodeBlockHeader)
 
-payloadWithOutputsBatch :: Env -> ChainId -> [DbHash PayloadHash] -> IO (Maybe [BlockPayloadWithOutputs])
-payloadWithOutputsBatch env (ChainId cid) hshes' = do
+payloadWithOutputsBatch :: Env -> ChainId -> M.Map (DbHash PayloadHash) Hash -> IO (Maybe (M.Map Hash BlockPayloadWithOutputs))
```

Comment:
Can we make this `Map (DbHash PayloadHash) a` and then return `[(a, BlockPayloadWithOutputs)]`?  That will be more generic.